docs/eove-his-interface.md

```diff
@@ -0,0 +1,423 @@
+    "module": {
+      "type": "vm150",
+      "serialNumber": "EO1500617140",
+      "apiVersion": "32.2",
+      "cpuVersion": "C150000702",
+      "powerVersion": "P150000400",
+      "gaugeVersion": "5",
+      "batteryType": "000000241701"
+    },
+    "station": {
+      "type": "eodisplay",
+      "applicationVersion": "3.2.0-dev.3",
+      "systemVersion": "eove-eodisplay-2.2.1",
+      "bootloaderVersion": "2017.12.0-EOVE_v1.2.2-g598a8e923"
+    }
+  }
+}
+```
+
+Note: ventilation module data are available only if the ventilation module is turned ON and inserted in the docking station.
+
+## Ventilation data
+
+The ventilation data are provided through subscription to channels.  
+
+The client sends `SUBSCRIBE` or `UNSUBSCRIBE` commands for a given channel.
+
+Available channels are:
+
+- `waveforms`: to receive a batch of waveforms
+- `monitorings`: to receive monitorings updates
+- `settings`: to receive settings updates for current ventilation mode
+- `alarms`: to receive alarms activations/deactivations
+- `ventilation`: to receive ventilation related information and updates
+
+The response to a subscription is an initial `snapshot`, followed by `patches` to update the data.
+
+When ventilaton module reconnects a snapshot is sent before any patch.
+Client can then replace all the state it has built so far.
+
+Snapshot/patch payloads include `newborn` information when patient type is pediatric.
+When patient type becomes adult, `newborn` information is set to `UNAVAILABLE` in patch's payload.
+Generally speaking any information missing from previous state in current state is set to `UNAVAILABLE` in patch's payload.
+This is a way to represent the absence of information in a incremental update.
+
+
+### Waveforms
+
+- prerequisite : communication started
+
+#### Subscription
+
+Client sends:
+
+```json
+{
+  "type": "SUBSCRIBE",
+  "payload": ["waveforms"],
+}
+```
+
+Upon success, clients receives:
+
+```json
+{ "type": "SUBSCRIBE_SUCCEEDED" }
+````
+
+#### Data 
+
+Waveforms can be build from a stream of samples that contain timestamp, pressure, flow, and volume. 
+A sample is a table with this format : `[number (epoch ms), number, number, number]`.
+
+For compacity and performance, samples are grouped in 12-sample chunks such as : 
+```json
+{
+  "type": "WAVEFORMS",
+  "payload": [
+    [ 1646909463640, 19.6, 26.7, 357 ],
+    [ 1646909463680, 19.8, 26.7, 375 ],
+    [ 1646909463720, 19.6, 26.7, 393 ],
+    [ 1646909463760, 19.6, 26.7, 411 ],
+    [ 1646909463800, 19.6, 26.7, 429 ],
+    [ 1646909463840, 19.5, 26.7, 447 ],
+    [ 1646909463880, 19.6, 26.7, 465 ],
+    [ 1646909463920, 19.5, 26.7, 483 ],
+    [ 1646909463960, 19.7, 26.7, 501 ],
+    [ 1646909463990, 19.5, 26.7, 519 ],
+    [ 1646909464040, 19.7, 26.7, 537 ],
+    [ 1646909464080, 19.7, 26.7, 555 ]
+  ]
+}
+```
+
+A sample is taken every 40ms by the ventilation module, which results in 12-sample chunks emitted every 480ms from the station.
```

Comment:
Talk about newborn patient type:
- 80 ms for patient or pediatric without newborn
- 40 ms for pediatric with newborn